test/dotcom/alerts/disruptions/subway_test.exs

```diff
@@ -79,20 +82,168 @@ defmodule Dotcom.Alerts.Disruptions.SubwayTest do
              } = future_disruptions()
     end
 
-    test "handles alert with more than one active_period" do
```

Comment:
Can we keep the existing tests as-is?

Reply:
(I just mean to not delete/rewrite them)

Reply:
This test can't be preserved exactly as-is, because an alert with more than one `active_period` won't make it all the way into the output. The new test (`"splits the active_periods of alerts with more than one into different buckets"`) is actually as close as I was able to get with two separate `active_period`'s if they don't get combined.

But I suppose I could also restore a test with a setup closer to this one where the assert is updated to match what will happen in the new world (`long_alert` won't wind up exactly as-is in the output, but its `active_period`'s will get combined).

Reply:
Thanks for explaining! It took me too long to realize that's what the first test was doing 😓

Reply:
[PTALA](https://github.com/mbta/dotcom/pull/2402/commits/dee78bd7903bd8db58f62b7f325da6e8e6ed14f3)

(Annoyingly, Git is rendering the diff weird, so it's showing the restored test as being fully added, and then its removal diff kind of piecemeal through the rest of the diff, but I did restore it as best I could.)